src/hops/RemoteDefinition.cs

```diff
@@ -279,6 +281,36 @@ void GetRemoteDescription()
             }
         }
 
+        public double GetDocumentTolerance()
+        {
+            Assembly assembly = Assembly.GetAssembly(typeof(Grasshopper.Utility));
```

Comment:
In this case, we are likely running in normal Rhino/Grasshopper and have access to a RhinoDoc. When that happens, we should just return the RhinoDoc tolerance.

Reply:
See above comment for simpler reflection code